RebuyFormsRating.Droid/FeedbackSheet.cs

```diff
@@ -86,16 +85,32 @@ void bv_click(object sender, EventArgs e)
         {
             var bv = sender as Android.Widget.Button;
             if (bv != null) {
+                var feedback = new RatingViewFeedback();
                 var s = bv.Text;
                 if (String.Equals(s, strSend)) {
                     var tv = (EditText)customFeedbackSheet?.FindViewById(Resource.Id.feedbackText);
+                    var mv = (EditText)customFeedbackSheet?.FindViewById(Resource.Id.feedbackMail);
+
+                    if (!string.IsNullOrEmpty(mv.EditableText.ToString()) && !isValidEmail(mv.EditableText.ToString())) {
+                        mv.SetError("Ungültige E-Mail-Adresse", null);
```

Comment:
mhh can we have here multi language support?

maybe we can as well set an attribute to a specific value and then it will take the different source file